SU-SWS/stanford_publication
# D8CORE-3281: Publications node

# READY FOR REVIEW

# Summary
- Styling the publication node page.
- Styles for the publication citation paragraph.

# Needed By (Date)
- 1/29/21

# Urgency
- high

# Steps to Test

1. Make sure you have the publications branch enabled.
2. Create at least 3 publications so it can fill up the Related. Make sure they have at least one of the same terms.
3. Create a basic page and added APA and Chicago style citations

# Affected Projects or Products
- Publications Module
- SBT

# Associated Issues and/or People
- D8CORE-3281
- D8CORE-3279

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
